src/components/Pattern/Pattern.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PatternWrapper } from './Pattern.styles';
+
+const Pattern = ({ color = 'midblue', size = 'M' }) => {
+  let svg;
+  let patternSize;
+  let patternColor;
+
+  // validate size prop ('M' if validation fails)
+  if (['L', 'M', 'S'].includes(size.toUpperCase())) {
+    patternSize = size.toUpperCase();
+  } else {
+    patternSize = 'M';
+  }
+
+  // validate color prop ('midblue' if validation fails)
+  if (
+    [
+      'lightpink',
+      'darkpink',
+      'mattepink',
+      'logopink',
+      'lightgrey',
+      'darkgrey',
+      'darkburgundy',
+      'orange',
+      'lightblue',
+      'midblue',
+      'darkblue'
+    ].includes(color.toLowerCase())
+  ) {
+    patternColor = color.toLowerCase();
+  } else {
+    patternColor = 'midblue';
+  }
```

Comment:
This color if statemant is unnessecary.

You can pass the color prop directly to the styled component.